packages/block-library/src/latest-posts/edit.js

```diff
@@ -72,47 +70,97 @@ class LatestPostsEdit extends Component {
 		this.isStillMounted = false;
 	}
 
-	toggleDisplayPostDate() {
-		const { displayPostDate } = this.props.attributes;
+	updateAttribute( name, value ) {
 		const { setAttributes } = this.props;
 
-		setAttributes( { displayPostDate: ! displayPostDate } );
+		setAttributes( { [ name ]: value } );
 	}
 
+// Difference between rangecontrol and querycontrol onNumberOfItemsChange??
+
 	render() {
 		const { attributes, setAttributes, latestPosts } = this.props;
 		const { categoriesList } = this.state;
-		const { displayPostDate, align, postLayout, columns, order, orderBy, categories, postsToShow } = attributes;
+		const { displayFullPost, displayPostContent, displayPostFeaturedImage, displayPostDate, displayPostAuthor, align, postLayout, columns, order, orderBy, categories, postsToShow, wordsToShow } = attributes;
```

Comment:
can we rename `wordsToShow` to `wordCount` or similar, indicating that it sets the number of words that will be displayed? named like this it suggests that we decide what/which wordsToShow.
on the same note `postsToShow ` has the same issue :)

Reply:
Sure, will do!

Reply:
I had actually already changed wordsToShow to excerptLength in the later commits which I think is even more descriptive